cmd/login/login.go

```diff
@@ -87,12 +89,18 @@ func login(cmd *cobra.Command, configuration *config.Configuration, envName stri
 		return errorUtils.NewWrappedError(ErrDecodingJwt, err)
 	}
 
-	selectedEnv, err := selectEnvironment(configuration.GetArmoryCloudAddr(), response.AccessToken, envName)
+	err = writeCredentialToFile(cli, parsedJwt, response)
+	if err != nil {
+		return err
+	}
+
+	CloudClient := configuration.NewClient(cli)
```

Comment:
any reason why "CloudClient" instead of cloudClient ?

Reply:
Nope, good catch. 👍